JavaScript/JavaScriptSDK/Initialization.ts

```diff
@@ -154,6 +154,9 @@ module Microsoft.ApplicationInsights {
             config.verboseLogging = Util.stringToBoolOrDefault(config.verboseLogging);
             config.emitLineDelimitedJson = Util.stringToBoolOrDefault(config.emitLineDelimitedJson);
             config.diagnosticLogInterval = config.diagnosticLogInterval || 10000;
+            config.autoTrackPageVisitTime = (config.autoTrackPageVisitTime !== undefined && config.autoTrackPageVisitTime !== null) ?
```

Comment:
I think "(config.autoTrackPageVisitTime !== undefined && config.autoTrackPageVisitTime !== null)" will be hadled by Util.stringToBoolOrDefault() so you don't need to do this check yourself